backend/app/channels/game_channel.rb

```diff
@@ -0,0 +1,22 @@
+class GameChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "game_channel_#{current_user.id}"
+  end
+  def unsubscribed
+    Game.leave_game(current_user) if current_user.present?
+  end
+
+  def joinGame(data)
+    Game.join(current_user) if current_user.present?
```

Comment:
If no `data` is passed, does this imply creating a new game?

Reply:
Yes, In Game join logic, As soon as person tries to join the game he will join a game where somebody is waiting him to join or a new game will be created